keyboards/uzu42/rev1/rev1.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "../uzu42.h"
+
+#include "quantum.h"
+
+#ifdef RGBLIGHT_ENABLE
+//rgb led driver
+#include "ws2812.h"
+#endif
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+#define LAYOUT( \
+  L00, L01, L02, L03, L04,           R00, R01, R02, R03, R04, \
+  L10, L11, L12, L13, L14,           R10, R11, R12, R13, R14, \
+  L20, L21, L22, L23, L24,           R20, R21, R22, R23, R24, \
+  L30, L31, L32, L33, L34, L35, R30, R31, R32, R33, R34, R35 \
+  ) \
+  { \
+    { L00, L01, L02, L03, L04 }, \
+    { L10, L11, L12, L13, L14 }, \
+    { L20, L21, L22, L23, L24 }, \
+    { L30, L31, L32, L33, L34, L35 }, \
+    { R04, R03, R02, R01, R00 }, \
+    { R14, R13, R12, R11, R10 }, \
+    { R24, R23, R22, R21, R20 }, \
+    { R35, R34, R33, R32, R31, R30 } \
+  }
+
+#define LAYOUT_kc( \
```

Comment:
Could this be removed? 

We're actively discouraging the use of `LAYOUT_kc`.  Not only does it not work well in the configurator, but it actually causes loads of confusion for newer uses, and additional work just to get things working.